content/uiux/concepts/stroke/stroke.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'Stroke'
+Description: 'A **stroke**, also referred to as a **border**, is a line that wraps around a layer.'
+Subjects:
+  - 'Web Design'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'UI'
+  - 'UX'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'intro-to-ui-ux'
+  - 'path/front-end-engineer-career-path'
+---
+
+In [design software](https://www.codecademy.com/resources/docs/uiux/design-software), the stroke properties can be updated to customize its style, size, and color.
+
+### Stroke Properties
+
+- A stroke can be solid or dashed. A **solid** stroke is an uninterrupted line. A **dashed** stroke consists of dashes or small lines spaced at uniform intervals.
+
+![Diagram that compares shapes with no stroke, solid stroke, and a dashed stroke.](https://static-assets.codecademy.com/Courses/intro-to-ui-and-ux/docs/Stroke.png)
```

Comment:
Could we move this image above the first bullet point to keep all the bullets together?